packages/components/pagination/__tests__/pagination.test.tsx

```diff
@@ -53,16 +48,10 @@ describe('Pagination', () => {
     })
     test('current-page defined while absence of current-page listener is invalid', () => {
       expect(console.warn).not.toHaveBeenCalled()
-      const wrapper = mount({
-        setup() {
-          return () => {
-            return h(Pagination, {
-              total: 100,
-              currentPage: 1,
-            })
-          }
-        },
-      })
+      const wrapper = mount(
+        <Pagination total={100} currentPage={1}></Pagination>
+      )
```

Comment:
Could you change the syntax to `mount(() => )` for consistency with other test statements?